src/stuf/templates/stuf/includes/security.xml

```diff
@@ -0,0 +1,18 @@
+{% if use_wss %}
```

Comment:
this was conditional for StUF-ZDS, but always present for StUF-BG (no conditional there). Could break something?

Reply:
Accepted risk